File: RedSheet.DbEntities/Models/vNanoRelationshipRequire.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vNanoRelationshipRequires", Schema = "dbo")]
	[Serializable]
	public partial class vNanoRelationshipRequire  
	{
 
	[Required] 
	[MaxLength(200)]
	public string ClassName { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int NanoRelationshipRequireId { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string NanoRelationshipRequireName { get; set; } 

	public vNanoRelationshipRequire()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-nano-relationship-require.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vNanoRelationshipRequire {
    constructor(vNanoRelationshipRequire?: vNanoRelationshipRequire )  {
        let properties = [ "className", "nanoRelationshipRequireId", "nanoRelationshipRequireName",];
        for (let property of properties)
            if (vNanoRelationshipRequire && vNanoRelationshipRequire[property])
                this[property] = vNanoRelationshipRequire[property];
    }
 
    @required()
    @maxLength(200)
	className : string =   undefined;
 
	nanoRelationshipRequireId : number =   0 ;
 
    @required()
    @maxLength(200)
	nanoRelationshipRequireName : string =   undefined;


}
